### PR TITLE
Add configurable options for oe-tools draw reg.poly.

### PR DESCRIPTION
**Pull request by adube**
This PR allows to configure the radius of the shapes created by the "draw regular polygon from click" directive within the `gmf-objecteditingtools` directive.

### CODE REVIEW

**Comment by adube** [comment]
Ready for review.

File: contribs/gmf/src/directives/objecteditingtools.js [hunk 1]
```diff
@@ -191,11 +192,16 @@ gmf.ObjecteditingtoolsController = function($scope, ngeoDecorateInteraction,
...
+    $injector.has('gmfObjectEditingToolsOptions') ?
+      $injector.get('gmfObjectEditingToolsOptions') : {});
+
   /**
    * @type {number}
    * @export
    */
-  this.triangleRadius = 100;
+  this.triangleRadius = oeToolsOptions.regularPolygonRadius !== undefined ?
+    oeToolsOptions.regularPolygonRadius : 100;
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/objecteditingtools.js]: Just a little point:  Can you add the default value like this:

https://github.com/camptocamp/ngeo/blob/699a903ba78334f01d3255458b74477764cc9576/contribs/gmf/src/services/thememanager.js#L12

I think that's more compact and we don't need to load `$injector`. (Don't do it if you see any problem about this.)

File: contribs/gmf/src/directives/objecteditingtools.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ gmf.module.directive('gmfObjecteditingtools', gmf.objecteditingtoolsDirective);
 
 
 /**
+ * @param {angular.$injector} $injector Main injector.
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/objecteditingtools.js]: Can you add a note about the existence of the configuration object `gmfx.ObjectEditingToolsOptions` in the description of this controller ?

Comment (adube) [inline contribs/gmf/src/directives/objecteditingtools.js]: fixed

**Pull request merged**